pkg/apis/acme/v1/types_issuer.go

```diff
@@ -518,43 +518,55 @@ type ACMEIssuerDNS01ProviderRoute53 struct {
 // ACMEIssuerDNS01ProviderAzureDNS is a structure containing the
 // configuration for Azure DNS
 type ACMEIssuerDNS01ProviderAzureDNS struct {
-	// if both this and ClientSecret are left unset MSI will be used
-	// +optional
-	ClientID string `json:"clientID,omitempty"`
```

Comment:
Please revert the fields back to their original order to make it easier to review the changes.